cmd/oceanbench/set.go

```diff
@@ -63,7 +64,7 @@ type quantity struct {
 
 // defaultQuantities provides a list of common quantities we might measure.
 // This was originally defined in the netreceiver code, see
-// bitbucket.org/ausocean/iotsvc/netreceiver/lists.py:420
+// github.com/ausocean/clientsvc/netreceiver/lists.py:420
```

Comment:
There's an error here, we didn't migrate this code so the link should remain the same. That being said, I don't think we need the reference to netreceiver, it doesn't provide any extra information.

Reply:
Fixed.